src/types/query-string-array-format.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Named type alias for the {arrayFormat} property of {QueryString.stringifyOptions}
+ */
+type QueryStringArrayFormat =
+    | "bracket"

```

Comment:
It has to be a type alias because in the interface from the `query-string`, it's directly using a union type of those specific strings.

Reply:
![image](https://user-images.githubusercontent.com/11774799/94840184-0bc8c300-03e6-11eb-8eab-f529118bfb3c.png)
Can you at least test it out? I pulled down your fork and swapped it with an enum - the build runs & the tests are passing. What are you seeing when you swap it with an enum?

Reply:
@brandongregoryscott for some reason I was getting type errors when I tried it. Just tried again though, and it worked, so I switched it over.

Reply:
AH, I think it wasn't working because in the parameter list, I had `arrayFormat: QueryStringArrayFormat = "index"` instead of `arrayFormat: QueryStringArrayFormat = QueryStringArrayFormat.Index`